Common/Data/Custom/CBOE/CBOE.cs

```diff
@@ -0,0 +1,123 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NodaTime;
+using QuantConnect.Data.Market;
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+
+namespace QuantConnect.Data.Custom.CBOE
+{
+    public class CBOE : TradeBar
```

Comment:
Is this intended to be processed  in the same spirit as custom data? If so, I think we'll want to update the constructor to set `DataType = MarketDataType.Base`.